cmd/terramate/cli/cli.go

```diff
@@ -321,26 +424,47 @@ func (c *cli) initStack(dirs []string) error {
 
 func (c *cli) listStacks(mgr *terramate.Manager, isChanged bool) ([]terramate.Entry, error) {
 	if isChanged {
+		log.Trace().
+			Str("action", "listStacks()").
+			Str("stack", c.wd()).
+			Msg("`Changed` flag was set. List changed stacks.")
```

Comment:
```suggestion
		log.Trace().
			Str("action", "listStacks()").
			Str("workingDir", c.wd()).
			Msg("`Changed` flag was set. List changed stacks.")
```

Rationale: c.wd() is not a stack